FueledUtils/UIExtensions.swift

```diff
@@ -37,6 +37,26 @@ public extension UILabel {
 		])
 		self.font = UIFont(descriptor: newFontDescriptor, size: self.font.pointSize)
 	}
+
+	public func setText(_ string: String?, animated: Bool) {
```

Comment:
…which would reduce the resulting code to 
```label.reactive.text <~ viewModel.text.observe(context: transitionContext(duration: 0.25))```

It would also make the transition more customisable through the context function.